fonz/cli.py

```diff
@@ -11,92 +14,163 @@ def wrapper(*args, **kwargs):
         try:
             return function(*args, **kwargs)
         except ValidationError as error:
-            logger.error(f"{error}\n")
```

Comment:
What gets displayed when this error occurs?

Reply:
Instead it prints a [header line](https://github.com/dbanalyticsco/Fonz/blob/feature/misc/fonz/connection.py#L289-L296) with the SQL error in `connection.py` before the exception is raised. I could move those error parameters into the `ValidationError` object and handle the printing in `cli.py:wrapper` if you think that would be cleaner.